styles.css

```diff
@@ -0,0 +1,378 @@
+  cursor: pointer;
+}
+
+.first-nav {
+  background-color: #272a31;
+}
+
+.first-nav .fab {
+  padding-top: 5px;
+}
+
+.first-nav a {
+  color: #fff !important;
+  font-size: 0.8rem;
+}
+
+.campaign a {
+  border: 2px solid red;
+  margin-top: 2px;
+  color: red;
+}
+
+.second-nav {
+  background-color: #fff;
+  padding: 10px 10px;
+}
+
+.second-nav a {
+  color: #5f6063;
+}
+
+.introduction {
+  background-image: url("./assets/main-background.png");
+  background-repeat: repeat-x;
+  background-position: center center;
+  z-index: 0;
+  margin-top: 0;
+  padding-right: 10rem;
+}
+
+.museum-history {
+  margin-right: 10rem;
+}
+
+.intro-greeting {
+  margin-top: 130px;
+  font-size: 50px;
+  font-weight: 300;
+}
+
+.intro-tag {
+  color: red;
+  margin-right: 10rem;
+  font-size: 55px;
+  font-weight: 900;
+}
+
+.intro-explanation {
+  align-items: center;
+  border: 4px solid rgb(255, 255, 255);
+  width: 631px;
+  background-color: rgb(247, 247, 248);
+}
+
+.intro-time {
+  font-size: 20px;
+  font-weight: 600;
+}
+
+.intro-footer {
+  font-size: 15px;
+  font-weight: 200;
+}
+
+h1 {
+  color: red !important;
+  font-size: large;
+}
+
+.program {
+  background-image: url("./assets/cardPatternBg.png");
+  background-color: rgb(39, 42, 50);
+  padding: 10vh 0;
+}
+
+.card {
+  border: none;
+}
+
+.program .card {
+  border: 2px solid transparent;
+  background-color: #fff1;
+  text-align: center;
+  margin: 1px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.program .card:hover {
+  border: 2px solid white;
+}
+
+.partner-images img {
+  width: 150px;
+  object-fit: contain;
+  filter: grayscale(100%);
+}
+
+.program .card img {
+  width: 60px;
+  margin: auto;
+}
+
+.program .card h5 {
+  color: #ec5242;
+}
+
+.centered-hr {
+  display: flex;
+  justify-content: center;
+}
+
+.program .card p {
+  color: #fff;
+  font-size: 0.9rem;
+}
+
+.program-title {
+  color: white;
+  text-align: center;
+}
+
+.program-schedule {
+  color: white;
+  text-align: center;
+  text-decoration: underline;
+}
+
+.partners {
+  background-color: #272a31;
+  padding: 7vh 0;
+}
+
+.partner-title {
+  text-align: center;
+  font-size: 24px;
+  color: #5f6063;
+}
+
+.partner-images {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+.footer {
+  background-color: #d3d3d3;
+  padding: 8vh 0;
+}
+
+hr {
+  background-color: red;
+  height: 2px !important;
+  width: 40px !important;
+  align-self: center;
+  text-align: center;
+  margin-top: 0;
+}
+
+.divider {
+  background-color: red;
+  height: 2px !important;
+  width: 40px !important;
+  align-content: center;
+}
+
+.guests-title {
+  text-align: center;
+}
+
+.guest-profession {
+  color: red;
+}
+
+.speaker {
+  display: none;
+}
+
+.btn-program {
+  display: none;
+}
+
+.btn-primary {
+  display: none;
+}
+
+.card-title {
+  font-weight: bolder;
+}
+
+.card-title,
+.card-description {
+  text-align: left;
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] It would be nice if your styling follows the **mobile-first** approach which means that your media queries should change the design from mobile to desktop, not the other way around. Kindy consider that 👍